images/builder/main.go

```diff
@@ -153,7 +170,7 @@ func runSingleJob(o options, jobName, uploaded, version string, subs map[string]
 type variants map[string]map[string]string
 
 func getVariants(o options) (variants, error) {
-	content, err := ioutil.ReadFile(path.Join(o.imageDirectory, "variants.yaml"))
+	content, err := ioutil.ReadFile("variants.yaml")
```

Comment:
Is this right? We should be looking for this in the config dir, but at this point we will be in the build dir.